### DISCUSSION TITLE
Visualization of spectra from previous experiments with DIANN viewer

### DISCUSSION THREAD

**Discussion by marthaspn** [vdemichev/DiaNN#1305] [Q&A]
Hello!

I would like to know if it is possible to reload a parquet file from a previous experiment in diann to visualise the spectra and if so, how I can achieve this.

Thanks in advance!

Best,
Martha

**Comment by vdemichev** [comment]
Hi Martha,

The 'Viewer' button configures things just based on the settings that you have in the GUI when you press it. So it does not need to be a recent experiment. So long as all the output has not been overwritten you can always launch the Viewer for it, just need to make sure the file names are correctly entered. If you ran the experiment as a pipeline, it will have a .pipeline file saved for each individual pipeline step in the respective output folder, so loading this file in the GUI will immediately give you the correct settings for the Viewer.

Best,
Vadim
